src/screens/Notification.js

```diff
@@ -193,6 +214,58 @@ return (
           </PropertyValue>
         </Property>
 
+        <Property name="notification.global.container">
+          <Description>
+            Any valid Box prop for the Global Notification container.
+          </Description>
+          <PropertyValue type="object">
+            <Example defaultValue>
+              {`
+  {
+  pad: {
+    horizontal: 'large',
+    vertical: 'xsmall',
+  },
+                `}
+            </Example>
+          </PropertyValue>
+        </Property>
+        <Property name="notification.direction">
```

Comment:
keep all .globals grouped together? Move this up alphabetically?

Reply:
Done, but curious about your thoughts of how the statuses are handled wrt `global` `toast` groupings.

Reply:
Like do they feel overkill or redundant? Can we create some sort of shorthand which references the properties?

Something like this?

```
notification.critical

object: {
  background: // any background properties,
  color: // any color properties,
  icon: <StatusCriticalSmall /> // any icon component or svg,
  ...
}
```